components/identity-event/org.wso2.carbon.identity.event/src/main/java/org/wso2/carbon/identity/event/IdentityEventConstants.java

```diff
@@ -86,6 +86,21 @@ private Event(){}
 
     }
 
+    /**
+     * Data Publishing Event Handler related constants triggered when a session is changed.
+     */
+    public enum EventName {
+
+        AUTHENTICATION_STEP_SUCCESS,
+        AUTHENTICATION_STEP_FAILURE,
+        AUTHENTICATION_SUCCESS,
+        AUTHENTICATION_FAILURE,
+        SESSION_CREATE,
+        SESSION_UPDATE,
+        SESSION_TERMINATE,
+
```

Comment:
Please remove the unnecessary black line

Reply:
Fixed with 15abde06a59a32d51f777778b9551910515589ca